### ISSUE TITLE
TabIndex order.

### ISSUE THREAD

**Issue by Sensetivity** [unclead/yii2-multiple-input#197]
![i604 cimgpsh_orig](https://user-images.githubusercontent.com/4289543/35517553-7705ccec-0517-11e8-9ee4-70c4105bc38f.png)
On Windows it`s work file. When I put to tab button it jump by next order: 1,2,3,4
On Mac OS it jump like this: 1,2,4,3.

**Comment by farhod111** [comment]
Hi Guys. Can I use this with Kartik's Select2 widet ?

**Comment by unclead** [comment]
@farhod111  have you this one https://github.com/unclead/yii2-multiple-input/wiki/Usage ?

**Comment by farhod111** [comment]
@unclead thanks for your answer. I try with this but widget didn't see data. I've used like this
```
<?= $form->field($model, 'schedule')->widget(MultipleInput::className(), [
'max' => 6,
'columns' => [
   [
      'name'  => 'group',
      'type'  => \kartik\date\Select2::className(),
      'title' => 'Day',
       'value' => function($data) {
            return $data['name'];
        },
       'items' => ArrayHelper::map(Group::find()->orderBy(['id' => SORT_DESC])->all(), 'id', 'name'),
    ],
    [
        'name' => 'language',
        'type' => 'dropDownList',
        'items' => StudentsForm::languages(),
        'title' => 'Language',
    ],
  ]
]
```
There is nothing in dropdowns data

**Comment by Sensetivity** [comment]
Can try put into 'options' sub-array

```
[
      'name'  => 'group',
      'type'  => \kartik\date\Select2::className(),
      'title' => 'Day',
       'value' => function($data) {
            return $data['name'];
        },
       'options' => [
'items' => ArrayHelper::map(Group::find()->orderBy(['id' => SORT_DESC])->all(), 'id', 'name'),
],
    ],
```

**Comment by elmanth** [comment]
use data instead items
`
[
      'name'  => 'group',
      'type'  => \kartik\date\Select2::className(),
      'title' => 'Day',
       'value' => function($data) {
            return $data['name'];
        },
       'options' => [
            'data' => ArrayHelper::map(Group::find()->orderBy(['id' => SORT_DESC])->all(), 'id', 'name'),
       ],
    ],
`

**Comment by farhod111** [comment]
Thank you! It works. My code is:
```
[
     'name'  => 'group',
     'type'  => Select2::className(),
     'title' => 'Day',
     'value' => function($data) {
           return $data['name'];
     },
     'options' => [
           'data' => ArrayHelper::map(Group::find()->orderBy(['id' => SORT_DESC])->all(), 'id', 'name'),
     ],
],
```

**Comment by Sensetivity** [comment]
Issue not about proble with data or arrays. Issue is on wrong tab-index on MacOS!

**Comment by unclead** [comment]
@Sensetivity  sorry, but i can't get the issue. 
Do you understand that the widget doesn't know about browser at all, it just renders HTML and provide some client side logic. 
In other words i don't see here an issue

**Comment by unclead** [comment]
@Sensetivity  i guess i understood the problem. You are talking about the lack of html `tabindex` property.
If you know how to fix it PR is always welcome.

**Comment by unclead** [comment]
@Sensetivity and so are you going to make a PR? 😄

**Comment by unclead** [comment]
fixed via 46bb2af5b801ff20b673a837b0c8825502343e2b
